src/main/java/org/update4j/service/DefaultBootstrap.java

```diff
@@ -129,6 +80,83 @@ public void main(List<String> args) throws Throwable {
 		}
 	}
 
+	void parseArgs(List<String> args){
+
+	    //using empty property manager to replace placeholders in commandline
+        ArrayList<String> props = new ArrayList<>(System.getenv().keySet());
+        for(Object o: System.getProperties().keySet()){
```

Comment:
Gotcha! This causes the `PS1` exception on build server, in turn driving you to change behavior for placeholders in system properties. `PropertyManager` is smart enough and you shouldn't have to add them manually. You can use the `null` for both parameters.

The real purpose of passing system properties to the constructor is to pre-resolve them so `implyPlaceholder()` knows if it and can use it to replace. For `resolvePlaceholder()` this is not needed.

Reply:
Using ``PropertyManager(null, null)`` and reverted the change in ``PropertyManager``.

Reply:
Also no need for the `props` list anymore.

Reply:
Done on next check in